src/settings/Settings.tsx

```diff
@@ -131,6 +131,14 @@ export interface ISetting {
 }
 
 export const SETTINGS: {[setting: string]: ISetting} = {
+    "feature_report_to_moderators": {
+        isFeature: true,
+        displayName: _td("Report to moderators prototype. " +
+            "In rooms that support moderation, the `report` button will let you report abuse to room moderators"),
+        supportedLevels: LEVELS_FEATURE,
+        default: false,
+        controller: new ReloadOnChangeController(),
```

Comment:
I don't think this is necessary given that this value is only read in a modal which cannot be visible at the same time as the Settings > Labs modal, this line can simply be omitted.